assets/javascripts/discourse/initializers/extend-for-assigns.js

```diff
@@ -15,6 +15,7 @@ import { iconHTML, iconNode } from "discourse-common/lib/icon-library";
 import discourseComputed from "discourse-common/utils/decorators";
 import I18n from "I18n";
 import BulkAssign from "../components/bulk-actions/assign-user";
+import TopicLevelAssignMenu from "../lib/topic-level-assign-menu";
```

Comment:
Maybe we could put this in component? It feels more like a component than a lib.

Reply:
Yeah, this makes sense. We usually put such classes into the _lib_ folder, but I agree that the _components_ folder is a better place for it, even though technically speaking it's not an Ember component. I moved it into `components/` and everything seem to work fine,